src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileupToBedPE.scala

```diff
@@ -0,0 +1,127 @@
+package com.fulcrumgenomics.sv.tools
+
+import com.fulcrumgenomics.commons.CommonsDef._
+import com.fulcrumgenomics.commons.io.Writer
+import com.fulcrumgenomics.commons.util.LazyLogging
+import com.fulcrumgenomics.sopt.{arg, clp}
+import com.fulcrumgenomics.sv.cmdline.{ClpGroups, SvTool}
+import com.fulcrumgenomics.sv.tools.AggregateSvPileupToBedPE.BedPE
+import com.fulcrumgenomics.sv.tools.AggregateSvPileupToBedPE.BedPE.BedPEWriter
+import com.fulcrumgenomics.util.{Io, Metric}
+import htsjdk.tribble.annotation.Strand
+
+import java.io.BufferedWriter
+
+
+@clp(group=ClpGroups.Utilities, description= "Convert the output of AggregateSvPileup to BEDPE.")
+class AggregateSvPileupToBedPE(
+  @arg(flag='i', doc="Input text file of aggregate pileups generated by AggregateSvPileup") input: FilePath,
+  @arg(flag='o', doc="Output text file of the aggregate pileups in BEDPE format.") output: FilePath,
+) extends SvTool with LazyLogging {
+
+  Io.assertReadable(input)
+  Io.assertCanWriteFile(output)
+
+  override def execute(): Unit = {
+    val reader = Metric.iterator[AggregatedBreakpointPileup](input)
+    val writer = BedPEWriter(output)
+
+    reader.map(BedPE.apply).foreach(writer.write)
+
+    writer.close()
+  }
+}
+
+/** Companion object for [[AggregateSvPileupToBedPE]]. */
+object AggregateSvPileupToBedPE {
+
+  /** The IGV-supported BEDPE file extension. */
+  val BedPEExtension: String = ".bedpe"
+
+  /** A simple BEDPE record as defined by `bedtools`:
+    *
+    * - https://bedtools.readthedocs.io/en/latest/content/general-usage.html#bedpe-format)
+    *
+    * Future compatibility could be implemented for supporting 10x flavored BEDPE files:
+    *
+    * - https://github.com/igvteam/igv/wiki/BedPE-Support
+    *
+    * Note that the field `score` is allowed to be a string per bedtools!
```

Comment:
'tis silly, and are we going to be ingesting strings?

Reply:
Probably not, but leaving it as String means I don't have to pick a numeric type and right now I'm mapping total templates to the score (integers). I can switch it to Integer because no matter what we do in the future, if it is a change, it will be an API breaking change anyway.